core/arch/arm/kernel/secure_partition.c

```diff
@@ -217,14 +219,175 @@ static TEE_Result sp_init_uuid(const TEE_UUID *uuid)
 {
 	TEE_Result res = TEE_SUCCESS;
 	struct sp_session *sess = NULL;
+	struct thread_smc_args args = {0};
 
 	res = sp_open_session(&sess,
 			      &open_sp_sessions,
 			      uuid);
 
+	ts_push_current_session(&sess->ts_sess);
+	sp_init_info(to_sp_ctx(sess->ts_sess.ctx), &args);
+	ts_pop_current_session();
+
+	if (sp_enter(&args, sess))
+		return FFA_ABORTED;
+
+	spmc_sp_msg_handler(&args, sess);
+
+	return res;
```

Comment:
can remove `res` and return `TEE_SUCCESS` here.

Reply:
Done